test/Spec.hs

```diff
@@ -43,6 +43,23 @@ testGetBug = testCase "getBug" go
       -- print bug'
       assertBool "Got bug ids" (isJust $ BZ.bugExternalBugs bug')
 
+testBugToTrackerData :: TestTree
+testBugToTrackerData = testCase "bugToTrackerData" go
+  where
+    go = do
+      bzSession <- bugzillaMockClient
+      Just bug' <- BZ.getBug bzSession 1791815
+      let td = toTrackerData bug'
+      assertBool
+        "Check extracted TrackerData"
+        ( case viaNonEmpty head td of
+            Just td' ->
+              tdIssueId td' == 1791815
+                && tdChangeUrl td' == "https://review.opendev.org/1717044"
+                && tdIssueUrl td' == "https://bugzilla.redhat.com/show_bug.cgi?id=1791815"
+            Nothing -> False
+        )
```

Comment:
Though, since we are using Hunit, it's better to use the assert operator to get better test failure:

```haskell
      let td = toTrackerData bug'
      case viaNonEmpty head td of                                                                                                                                                                                                                                                                                                                                                                                                          
          Just td' ->                                                                                                                                                                                                                                                                                                                                                                                                                        
            sequence_                                                                                                                                                                                                                                                                                                                                                                                                                        
              [ tdIssueId td' @=? 1791815,                                                                                                                                                                                                                                                                                                                                                                                                   
                tdChangeUrl td' @=? "https://review.opendev.org/1717044",                                                                                                                                                                                                                                                                                                                                                                    
                tdIssueUrl td' @=? "https://bugzilla.redhat.com/show_bug.cgi?id=1791815"                                                                                                                                                                                                                                                                                                                                                     
              ]                                                                                                                                                                                                                                                                                                                                                                                                                              
          Nothing -> assertBool "No external bugs found" False                                                                                                                                                                                                                                                                                                                                                                               
```